cohorts/survival.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
```

Comment:
Cool, I think it would be good to do that, with some standards on colors/size but this seems good for now